system/assets/js/admin/specific/datamanager/object/batchUpdateButton.js

```diff
@@ -21,4 +21,25 @@
 		} );
 	}
 
+	// select the target node
+	var target = document.querySelector('table');
+
+	// create an observer instance
+	var observer = new MutationObserver(function(mutations) {
+		mutations.forEach(function(mutation) {
+			if ( mutation.type === 'childList' ) {
+				console.log( "mutate" );
+				var $listingTable  = $( '.object-listing-table' );
+				var $selectAllCBox = $listingTable.find( "th input:checkbox" );
+				$selectAllCBox.click();
```

Comment:
Do we really want to click it any time there is a change in the table? If it is _unchecked_, does it then become selected? I think we need logic to only uncheck it when it is already checked.